nx-X11/programs/Xserver/hw/nxagent/Reconnect.c

```diff
@@ -438,22 +438,10 @@ Bool nxagentReconnectSession(void)
 
   if (nxagentKeyboard != NULL)
   {
-    int size;
+    nxagentOldKeyboard = strndup(nxagentKeyboard, strlen(nxagentKeyboard));
+    /* FIXME: check if nxagentOldKeyboard == NULL and abort */
```

Comment:
Well, I have not found a quick way to abort cleanly here. Besides: This is the reconnect part, do we really want to kill the session in that case?